src/test/resources/pages/newlyJoinedInstructorStudentFeedbackSubmissionEdit.html

```diff
@@ -5,12 +5,6 @@
     Submit Feedback
   </h1>
   <br>
-  <div id="student-motd-wrapper">
-    <input id="motd-url" type="hidden" value="${studentmotd.url}">
-    <script defer="" src="/js/studentMotd.js" type="text/javascript">
-    </script>
-    ${studentmotd.container}
-  </div>
```

Comment:
Should this block be removed? Seems like it is unrelated to the changes you are making.